src/openapi/streaming/streaming.js

```diff
@@ -174,12 +189,18 @@ function retryConnection() {
 function onConnectionStateChanged(nextState) {
     this.connectionState = nextState;
 
-    const connectionTransport = this.connection.getTransport();
-    log.debug(LOG_AREA, 'Connection state changed', {
-        changedTo: this.READABLE_CONNECTION_STATE_MAP[this.connectionState],
-        mechanism: connectionTransport && connectionTransport.name,
-        reconnecting: this.reconnecting,
-    });
+    const connectionTransport = this.getActiveTransportName();
+    log.info(

```

Comment:
This is going to generate alot of logs - maybe too many. can we just log within signalr core? or make persist depend on signalrcore being enabled for now or something?

Reply:
ok
will persist depend on signalrcore being enabled